diff --git a/feeds/packages/libs/libxcrypt/Makefile b/feeds/packages/libs/libxcrypt/Makefile
index 0000000..1111111 100644
--- a/feeds/packages/libs/libxcrypt/Makefile
+++ b/feeds/packages/libs/libxcrypt/Makefile
@@ -42,7 +42,7 @@
 	--disable-obsolete-api \
 	--enable-hashes=solaris \
 	--with-pic
-
+TARGET_CFLAGS += -Wno-error=format-nonliteral
 define Build/InstallDev
 	$(INSTALL_DIR) $(1)/usr/include
 	$(CP) $(PKG_INSTALL_DIR)/usr/include/*.h $(1)/usr/include/
